### DISCUSSION TITLE
Why is there that gray border under the textarea?

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#6970] [Help]
### REPL: https://play.tailwindcss.com/52JZTEbfF0

![image](https://user-images.githubusercontent.com/41120635/148644208-7d9c4782-4b3d-4f16-a94e-fc1a50ab5286.png)

Why is there that gray border under the textarea?

```html
<div class="bg-gray-100 px-10">
  <div>Hello</div>
  <textarea class="w-full"></textarea>
</div>
```

**Comment by NovusOpus** [comment]
~~It's because of [Preflight(Specifically Border Style Reset)](https://tailwindcss.com/docs/preflight#border-styles-are-reset-globally).~~

**Update:** 
Seems like you mean the margin/gap under the textarea, This also happens with plain CSS not only with Tailwind, and the reason is perfectly explained by [@adamwathan in his reply below](https://github.com/tailwindlabs/tailwindcss/discussions/6970#discussioncomment-1933228).

There's a few ways to remove it, here's a couple of them:

1. adding `block`(display:block) class to the textarea.
2. adding the `align-middle` or `align-bottom` class to the textarea.

**Reply by frederikhors** [reply]
Is there a way to fix?

**Reply by frederikhors** [reply]
I don't want that border becasue if I want it then I would use `mb-*`.

**Reply by NovusOpus** [reply]
I updated the original reply with some ways to remove the margin/gap under the textarea.

**Reply by frederikhors** [reply]
@NovusOpus I still don't understand why is this happening. With `block` it works, but it's still something strange to me. I'm also using https://github.com/tailwindlabs/tailwindcss-forms with the same problem.

**Comment by frederikhors** [comment]
@adamwathan I'm using https://github.com/tailwindlabs/tailwindcss-forms but the border is still there. Is this the normal way to go?

**Reply by adamwathan** [reply]
Hey! It’s because textareas are inline elements and are therefore affected by line height. Set it to block. This isn’t Tailwind specific, can observe this same behavior without Tailwind. Would recommend studying some CSS fundamentals to get a better understanding of things like this 👍🏻

**Reply by adamwathan** [reply]
Here’s a demo that will explain it, you can see the textarea lines up with the baseline of the text next to it:

https://play.tailwindcss.com/yqHcQg1HGv

The gap below is because of the space reserved for the descenders of the text at the current line height.
